frontend/src/components/nav.js

```diff
@@ -73,6 +74,7 @@ class NavigationBar extends Component {
       Router.push('/dashboard')
 
       this.setState({
+        isDirector: result.is_director,
```

Comment:
should we still be using `validateKey` if we have the server integrated? Just wondering

Reply:
ok sounds good

what do you mean by `until the backend is implemented` though? What is there left to do? Just want to keep an eye on the blockers for me and @chyku

Reply:
The PI's server requires a tokenId in order to retrieve a profile, so we need to create a table to hold the tokens for the users so we can get each user and their role. The backend is basically making the `get` endpoint for the tokens